### PR TITLE
Public notices updates

### PR DESCRIPTION
**Pull request by fionawhim**
- Adds Rollbar support
 - Adds a BUILD_ID checker to see if the code has changed
 - Updates to the v2 endpoint format
 - Fixes exponent bug (use Math.pow, not ^)
 - Moves static assets so that Next will serve them

### CODE REVIEW

File: services-js/public-notices/src/components/LastUpdated.tsx [hunk 1]
```diff
@@ -18,7 +18,7 @@ export default function LastUpdated({
   //January 7, 2019 1:51PM
   const lastUpdatedString = lastUpdated
     ? `Last updated: ${format(lastUpdated, 'MMMM D, YYYY h:mmA')}`
-    : '';
+    : 'Loading…';
```
Comment (jessicamarcus) [inline services-js/public-notices/src/components/LastUpdated.tsx:21]: 👍

File: services-js/public-notices/src/components/NoticeDetail.tsx [hunk 2]
```diff
@@ -246,12 +254,8 @@ export default class NoticeDetail extends React.Component<Props> {
                     <span className="notice-canceled">Canceled</span>
                   ) : (
                     <>
-                      <div
-                        dangerouslySetInnerHTML={{ __html: notice.notice_date }}
-                      />
-                      <div
-                        dangerouslySetInnerHTML={{ __html: notice.notice_time }}
-                      />
+                      <div>{notice.notice_date}</div>
```
Comment (jessicamarcus) [inline services-js/public-notices/src/components/NoticeDetail.tsx:257]: Glad to see those `dangerouslySetInnerHTML`s go!

File: services-js/public-notices/src/model/Notice.ts [hunk 3]
```diff
@@ -1,10 +1,10 @@
 /**
  * Type of a public notice from the Boston.gov public notices v1 API:
  *
- * https://www.boston.gov/api/v1/public-notices
+ * https://www.boston.gov/api/v2/public-notices
  */
 export default interface Notice {
-  is_testimony: '';
+  is_testimony: '0' | '1';
```
Comment (jessicamarcus) [inline services-js/public-notices/src/model/Notice.ts:7]: Could this become a boolean value instead?

**Review by jessicamarcus — APPROVED** [review]

File: services-js/public-notices/src/model/Notice.ts (hunk 3 above)
Comment (fionawhim) [reply services-js/public-notices/src/model/Notice.ts:7]: It’s not straightforward to, due to the way the data is getting pulled out of Drupal.

Comment (jessicamarcus) [reply services-js/public-notices/src/model/Notice.ts:7]: 👍 figured there must have been a good reason, but wanted to ask!

**Pull request merged**